crates/libtiny_tui/src/key_map.rs

```diff
@@ -357,13 +359,18 @@ fn deser_keymap() {
       tab_goto: 1
     a: 
       input: b
+    b:
+      run_command: clear
```

Comment:
I'd drop "run" as well as there's nothing else you can do with a command other than to run, so it's not helpful.

I'll do it after the merge.